### PR TITLE
Files - add media, source properties, & recordings special folder

### PR DESCRIPTION
**Pull request by elmazdremdzhe**

### CODE REVIEW

File: api-reference/beta/resources/driveItemSource.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+### driveItemSourceApplication enumeration
+
+| Value               | Description                                       |
+|:--------------------|:--------------------------------------------------|
+| teams               | The application is Teams.                         |
+| yammer              | The application is Yammer.                        |
+| sharePoint          | The application is SharePoint.                    |
+| oneDrive            | The application is OneDrive.                      |
+| stream              | The application is Stream.                        |
+| powerPoint          | The application is PowerPoint                     |
```
Comment (MarcMroz) [inline api-reference/beta/resources/driveItemSource.md:51]: Check this enum with what's in code. I think "Office" was added that should be listed here.

File: api-reference/beta/resources/media.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  "isTranscriptionShown" : true,
+  "mediaSource": { "@odata.type": "microsoft.graph.mediaSource" }
+}
+```
+
+## Properties
+
+| Property                 | Type                  | Description                                                                                                   |
+| :----------------------- | :-------------------- | :------------------------------------------------------------------------------------------------------------ 
+| **isTranscriptionShown** | boolean               | Indicates whether the transcription is shown. Read-Write.                                                     |
```
Comment (MarcMroz) [inline api-reference/beta/resources/media.md:35]: Can we change this to:
"If a file has a transcript, this setting controls if the closed captions / transcription for the media file should be shown to people during viewing. Read-Write.

File: api-reference/beta/resources/driveItemSource.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+author: MarcMroz
+description: "The driveItemSource contains metadata about the source application that the drive item was created in."
+ms.author: brcrowne
+ms.date: 07/15/2020
+title: DriveItemSource
```
Comment (Lauragra) [inline api-reference/beta/resources/driveItemSource.md:6]: The title needs to be "driveItemSource resource type".

File: api-reference/beta/resources/driveItemSource.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+localization_priority: Normal
+doc_type: resourcePageType
+ms.prod: "sharepoint"
```
Comment (Lauragra) [inline api-reference/beta/resources/driveItemSource.md:9]: The ms.prod value is "sites-and-lists".

File: api-reference/beta/resources/driveItemSource.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+---
+# DriveItemSource facet
```
Comment (Lauragra) [inline api-reference/beta/resources/driveItemSource.md:11]: The H1 title should be "driveItemSource resource type".

File: api-reference/beta/resources/driveItemSource.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+The **DriveItemSource** resource contains metadata about the source of the drive item.
```
Comment (Lauragra) [inline api-reference/beta/resources/driveItemSource.md:13]: Note that resource names in Microsoft Graph are all lower camel case, so the resource name needs to be **driveItemSource**.

File: api-reference/beta/resources/driveItemSource.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+}-->
+
+```json
+{
+  "application": "string",
+  "externalId" : "string"
+}
+```
+
+## Properties
```
Comment (Lauragra) [inline api-reference/beta/resources/driveItemSource.md:35]: Please follow the defined topic structure for resource topics. The Properties H2 section comes before the JSON representation.

File: api-reference/beta/resources/driveItemSource.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+| Property                 | Type                       | Description                                                                                      |
+| :----------------------- | :------------------------  | :----------------------------------------------------------------------------------------------- |
+| **application**          | driveItemSourceApplication | Enumeration value that indicates the source application where the file was created.              |
+| **externalId**           | string                     | The external identifier for the drive item from the source.                                      |
+
+### driveItemSourceApplication enumeration
```
Comment (Lauragra) [inline api-reference/beta/resources/driveItemSource.md:42]: The title here needs to be "driveItemSourceApplication values".

File: api-reference/beta/resources/driveItemSource.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+| teams               | The application is Teams.                         |
+| yammer              | The application is Yammer.                        |
+| sharePoint          | The application is SharePoint.                    |
+| oneDrive            | The application is OneDrive.                      |
+| stream              | The application is Stream.                        |
+| powerPoint          | The application is PowerPoint                     |
+| office              | The application is Office                         |
+| unknownFutureValue  | Marker value for future compatibility.            |
+
+## Remarks
```
Comment (Lauragra) [inline api-reference/beta/resources/driveItemSource.md:55]: This Remarks section is not standard even though some other SharePoint topics also include it. You can make this a ## See also section if it's just for a cross-reference. Otherwise, you can add the link to the summary section.

File: api-reference/beta/resources/driveItemSource.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+For more information about the facets on a DriveItem, see [DriveItem](driveitem.md).
```
Comment (Lauragra) [inline api-reference/beta/resources/driveItemSource.md:57]: Lower camel case -> driveItem.

File: api-reference/beta/api/drive-get-specialfolder.md [hunk 11]
```diff
@@ -1,10 +1,11 @@
 ---
 author: JeremyKelley
 description: "Use the special collection to access a special folder by name."
-ms.date: 09/10/2017
+ms.date: 07/15/2020
 title: Get Special Folders
 localization_priority: Normal
 ms.prod: "sharepoint"
+ms.author: brcrowne
```
Comment (Lauragra) [inline api-reference/beta/api/drive-get-specialfolder.md:8]: Please remove ms.author, we don't use this attribute in the Microsoft Graph docs.

File: api-reference/beta/resources/driveItemSource.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
+---
+author: MarcMroz
+description: "The driveItemSource contains metadata about the source application that the drive item was created in."
+ms.author: brcrowne
```
Comment (Lauragra) [inline api-reference/beta/resources/driveItemSource.md:4]: Remove ms.author and ms.date. Why would the two author values be different?

File: api-reference/beta/resources/media.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
+---
+author: MarcMroz
+description: "The media resource contains metadata about the media (audio or video) drive item."
+ms.author: brcrowne
+ms.date: 07/15/2020
+title: Media
```
Comment (Lauragra) [inline api-reference/beta/resources/media.md:6]: Please make all the same updates to the metadata mentioned previously. The title needs to be "Media resource type".

File: api-reference/beta/resources/media.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+```
+
+## Properties
+
+| Property                 | Type                  | Description                                                                                                   |
+| :----------------------- | :-------------------- | :------------------------------------------------------------------------------------------------------------ 
+| **isTranscriptionShown** | boolean               | If a file has a transcript, this setting controls if the closed captions / transcription for the media file should be shown to people during viewing. Read-Write.                                                    |
+| **mediaSource**          | [mediaSource](mediaSource.md)         | Information about the source of media. Read-only.                                                             | 
+
+## Remarks
```
Comment (Lauragra) [inline api-reference/beta/resources/media.md:38]: Remove, or make this a ## See also section.

File: api-reference/beta/resources/media.md [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+}-->
+
+```json
+{
+  "isTranscriptionShown" : true,
+  "mediaSource": { "@odata.type": "microsoft.graph.mediaSource" }
+}
+```
+
+## Properties
```
Comment (Lauragra) [inline api-reference/beta/resources/media.md:31]: Move the Properties to come before JSON representation.

File: api-reference/beta/resources/mediaSource.md [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+author: MarcMroz
+description: "The mediaSource resource contains metadata about source of the media (audio or video) drive item."
+ms.author: brcrowne
+ms.date: 07/15/2020
+title: MediaSource
+localization_priority: Normal
+ms.prod: "sharepoint"
+doc_type: "resourcePageType"
+---
+# MediaSource facet
```
Comment (Lauragra) [inline api-reference/beta/resources/mediaSource.md:11]: "mediaSource resource type"

File: api-reference/beta/resources/mediaSource.md [hunk 17]
```diff
@@ -0,0 +1,61 @@
...
+
+The **MediaSource** resource contains metadata about the source of media (audio or video) drive item.
```
Comment (Lauragra) [inline api-reference/beta/resources/mediaSource.md:13]: MediaSource -> mediaSource

File: api-reference/beta/resources/driveItemSource.md (hunk 6 above)
Comment (Lauragra) [reply api-reference/beta/resources/driveItemSource.md:13]: The description can just start with a verb: "Contains metadata...."

File: api-reference/beta/resources/media.md [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+ms.author: brcrowne
+ms.date: 07/15/2020
+title: Media
+localization_priority: Normal
+doc_type: resourcePageType
+ms.prod: "sharepoint"
+---
+# Media resouce type
+
+The **Media** resource contains metadata about the media (audio or video) drive item.
```
Comment (Lauragra) [inline api-reference/beta/resources/media.md]: Revise to be "Contains metadata...".

File: api-reference/beta/resources/mediaSource.md [hunk 19]
```diff
@@ -0,0 +1,61 @@
+---
+author: MarcMroz
```
Comment (Lauragra) [inline api-reference/beta/resources/mediaSource.md:2]: Update the metadata.

File: api-reference/beta/resources/mediaSource.md [hunk 20]
```diff
@@ -0,0 +1,61 @@
...
+  "@odata.type": "microsoft.graph.mediaSource"
+}-->
+
+```json
+{
+  "contentCategory" : "string"
+}
+```
+
+## Properties
```
Comment (Lauragra) [inline api-reference/beta/resources/mediaSource.md:33]: Move to come before JSON representation.

File: api-reference/beta/resources/mediaSource.md [hunk 21]
```diff
@@ -0,0 +1,61 @@
...
+
+| Property                 | Type                       | Description                                                                                      |
+| :----------------------- | :------------------------  | :----------------------------------------------------------------------------------------------- |
+| **contentCategory**      | mediaSourceContentCategory | Enumeration value that indicates the media content category.                                     |
+
+### mediaSourceContentCategory enumeration
```
Comment (Lauragra) [inline api-reference/beta/resources/mediaSource.md:39]: "mediaSourceContentCategory values"

File: api-reference/beta/resources/media.md [hunk 22]
```diff
@@ -0,0 +1,51 @@
...
+  "isTranscriptionShown" : true,
+  "mediaSource": { "@odata.type": "microsoft.graph.mediaSource" }
+}
+```
+
+## Properties
+
+| Property                 | Type                  | Description                                                                                                   |
+| :----------------------- | :-------------------- | :------------------------------------------------------------------------------------------------------------ 
+| **isTranscriptionShown** | boolean               | If a file has a transcript, this setting controls if the closed captions / transcription for the media file should be shown to people during viewing. Read-Write.                                                    |
```
Comment (Lauragra) [inline api-reference/beta/resources/media.md:35]: boolean -> Boolean

File: api-reference/beta/resources/mediaSource.md [hunk 23]
```diff
@@ -0,0 +1,61 @@
...
+
+| Value               | Description                                         |
+|:------------------- |:----------------------------------------------------|
+| meeting             | The media is a Meeting.                             |
```
Comment (Lauragra) [inline api-reference/beta/resources/mediaSource.md:43]: These meeting types are not proper nouns, please lowercase them.

File: api-reference/beta/resources/specialfolder.md [hunk 24]
```diff
@@ -49,15 +50,9 @@ If a user deletes one, it is recreated when written to again.
...
+[!INCLUDE [files-special-folder-list](../includes/files-special-folder-list.md)]
 
-| Name        | Folder id    | Description                                                              |
-|:------------|:-------------|:-------------------------------------------------------------------------|
-| App Root    | `approot`    | The application's personal folder. Usually in `/Apps/{Application Name}` |
-| Camera Roll | `cameraroll` | The Camera Roll Backup folder. Not available in OneDrive for Business.   |
-| Documents   | `documents`  | The Documents folder.                                                    |
-| Music       | `music`      | The Music folder. Not available in OneDrive for Business.                |
-| Photos      | `photos`     | The Photos folder.                                                       |
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
```
Comment (Lauragra) [inline api-reference/beta/resources/specialfolder.md:55]: Why are you adding the beta disclaimer here?

File: api-reference/beta/resources/specialfolder.md [hunk 25]
```diff
@@ -49,15 +50,9 @@ If a user deletes one, it is recreated when written to again.
 
 ## Special folders
```
Comment (Lauragra) [inline api-reference/beta/resources/specialfolder.md:51]: This is a non-standard H2 for a resource topic. Please remove. Can the content from the Includes file be part of the summary description? Please also make the same updates to this topic to conform to standards - metadata, casing, topic structure.

File: api-reference/beta/resources/driveItemSource.md [hunk 26]
```diff
@@ -0,0 +1,67 @@
+---
```
Comment (Lauragra) [inline api-reference/beta/resources/driveItemSource.md:1]: File names should be all lowercase. In the future, it will be helpful for you to use the stub generator tool to output your files to ensure that they are consistent with the standards.

File: changelog/Microsoft.FileServices.json [hunk 27]
```diff
@@ -1,5 +1,55 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the [mediaSource](https://docs.microsoft.com/en-us/graph/api/resources/mediaSource?view=graph-rest-beta) resource type.",
+          "Target": "mediaSource"
+        }
+      ],
+      "Id": "f67638e8-7659-11eb-9758-816d298b468f",
+      "Cloud": "prd",
+      "Version": "beta",
+      "CreatedDateTime": "2021-02-24T04:37:12.404Z",
+      "WorkloadArea": "Files",
```
Comment (Lauragra) [inline changelog/Microsoft.FileServices.json:50]: Is this Files, or Sites and lists?

File: concepts/changelog.md [hunk 28]
```diff
@@ -129,6 +129,13 @@ For a summary of the value of these API changes, as well as recent tools, compon
 |Addition|beta|Added the **plannerContainerType** enumeration type.|
 |Change|beta| Deprecated the **owner** property on [plannerPlan](/graph/api/resources/plannerPlan?view=graph-rest-beta&preserve-view=true) resource.|
 
+### Files
```
Comment (Lauragra) [inline concepts/changelog.md:132]: FYI, we no longer update the changelog.md file. That topic has been replaced by the filterable changelog experience on the developer site. You only need to include the JSON file update for the changelog entry.

File: api-reference/beta/resources/specialfolder.md (hunk 24 above)
Comment (elmazdremdzhe) [reply api-reference/beta/resources/specialfolder.md:55]: I think the disclaimer was intended for the "recordings" special folder type. Removed from here

File: changelog/Microsoft.FileServices.json (hunk 27 above)
Comment (elmazdremdzhe) [reply changelog/Microsoft.FileServices.json:50]: this is Files

**Comment by lramosvea** [comment]
@elmazdremdzhe please add the "content review" label once you have addressed Laura's comments

File: api-reference/beta/resources/driveItemSource.md (hunk 26 above)
Comment (elmazdremdzhe) [reply api-reference/beta/resources/driveItemSource.md:1]: I tried to change the case, but unsuccessfully - git changes in files are not case-sensitive

File: api-reference/beta/resources/mediaSource.md (hunk 23 above)
Comment (elmazdremdzhe) [reply api-reference/beta/resources/mediaSource.md]: added lower case

File: changelog/Microsoft.FileServices.json (hunk 27 above)
Comment (Lauragra) [reply changelog/Microsoft.FileServices.json:50]: The topics you're updating in this PR have sites-and-lists for the ms.prod value, not files. Are you sure that you don't want the changelog to appear under Sites and lists?

File: concepts/changelog.md [hunk 29]
```diff
@@ -32,32 +32,61 @@ For a summary of the value of these API changes, as well as recent tools, compon
 | Addition | beta | Added the **uniqueName** property to the [application](https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-beta) resource.|
 
 
+### Education
```
Comment (Lauragra) [inline concepts/changelog.md:35]: Please remove this file from the PR - it's a legacy file that we no longer update. I'm not sure where these unrelated changes are coming from. Going forward we only need to update the JSON file in the changelog folder.

File: api-reference/beta/api/driveitem-createuploadsession.md [hunk 30]
```diff
@@ -5,6 +5,7 @@ ms.date: 09/10/2017
 title: Resumable file upload
```
Comment (Lauragra) [inline api-reference/beta/api/driveitem-createuploadsession.md:5]: The title and the H1 heading should match. In this case, both should be "driveItem: createUploadSession", per our file naming guidelines for action topics.

File: api-reference/beta/resources/driveitem.md [hunk 31]
```diff
@@ -1,10 +1,11 @@
 ---
 author: JeremyKelley
 description: "The driveItem resource represents a file, folder, or other item stored in a drive."
-ms.date: 09/10/2017
+ms.date: 07/15/2020
```
Comment (Lauragra) [inline api-reference/beta/resources/driveitem.md:4]: Please remove this and ms.author and update the ms.prod value - ideally, all topics for a workload will have a consistent value so you can easily filter to your topics in the data reports.

File: api-reference/beta/resources/driveitemsource.md [hunk 32]
```diff
@@ -0,0 +1,65 @@
...
+}-->
+
+```json
+{
+  "application": "string",
+  "externalId" : "string"
+}
+```
+
+### driveItemSourceApplication values
```
Comment (Lauragra) [inline api-reference/beta/resources/driveitemsource.md:40]: This H3 should be a subsection of the Properties H2 instead of JSON representation.

File: api-reference/beta/resources/driveitemuploadableproperties.md [hunk 33]
```diff
@@ -3,7 +3,8 @@ title: "driveItemUploadableProperties resource type"
 description: "The driveItemUploadableProperties resource represents an item being uploaded when creating an upload session."
 localization_priority: Normal
 author: "JeremyKelley"
-ms.prod: ""
+ms.prod: "sharepoint"
+ms.author: brcrowne
```
Comment (Lauragra) [inline api-reference/beta/resources/driveitemuploadableproperties.md:7]: Remove ms.author, update ms.prod.

File: api-reference/beta/resources/mediasource.md [hunk 34]
```diff
@@ -0,0 +1,59 @@
...
+  "@odata.type": "microsoft.graph.mediaSource"
+}-->
+
+```json
+{
+  "contentCategory" : "string"
+}
+```
+
+### mediaSourceContentCategory values
```
Comment (Lauragra) [inline api-reference/beta/resources/mediasource.md:37]: Move the H3 to the Properties section.

**Review by Lauragra — APPROVED** [review]
